src/languages/es.ts

```diff
@@ -4449,6 +4450,27 @@ const translations = {
                 },
             },
         },
+        downgrade: {
```

Comment:
Ask to confirm here: https://expensify.slack.com/archives/C01GTK53T8Q/p1734603543349989

Reply:
@jayeshmangwani Nice, thanks for pointing out that